flatpak/com.mitchellh.ghostty.Devel.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.mitchellh.ghostty.Devel
+runtime: org.gnome.Platform
+runtime-version: "47" # maybe use gnome-nightly master?
```

Comment:
I'm down for this pending what @jcollie and @pluiedev think

Reply:
Hmm, if developing Gnome/GTK were the primary object sure, but since this is about developing Ghostty we should probably stick to a released version of Gnome. 48 is very close to release so if there was like a 48-rc channel I'd be OK with that.

Reply:
I agree. 48-rc/beta/etc if possible, 47 otherwise

Reply:
If the goal is to reuse this manifest for Flathub, using RCs might be a problem as one wouldn't want a stable release of the Flatpak package to depend on RC/beta releases.

It might be better to just depend on a stable release and bump it every 3-6 months or something along those lines.

Reply:
This one is a development manifest (`.Devel`), I don't think it will ever make it Flathub.

The regular one will still use stable.

I guess with the current consensus I'll remove the comment since GNOME doesn't publish rc runtimes.

Reply:
Oops, I completely missed the `.Devel` bit. Sorry for the noise! :man_facepalming: